src/openforms/prefill/contrib/family_members/plugin.py

```diff
@@ -0,0 +1,78 @@
+import logging
+
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from openforms.authentication.service import AuthAttribute
+from openforms.contrib.family_members.constants import FamilyMembersDataAPIChoices
+from openforms.contrib.family_members.models import FamilyMembersTypeConfig
+from openforms.contrib.haal_centraal.checks import (
+    check_config as check_haal_centraal_config,
+)
+from openforms.submissions.models import Submission
+from openforms.typing import JSONEncodable
+from stuf.stuf_bg.checks import check_config as check_stuf_bg_config
+
+from ...base import BasePlugin
+from ...registry import register
+from .api.serializers import FamilyMembersOptionsSerializer
+from .haal_centraal import get_data_from_haal_centraal
+from .stuf_bg import get_data_from_stuf_bg
+from .typing import FamilyMembersChildOptions, FamilyMembersPartnerOptions
+
+logger = logging.getLogger(__name__)
+
+
+PLUGIN_IDENTIFIER = "family_members"
+
+
+@register(PLUGIN_IDENTIFIER)
+class FamilyMembersPrefill(
```

Comment:
As mentioned during standup as well, I chose to go for one prefill plugin because to me it has to do with one thing (retrieving person's information) that can be achieved by using two different sources. Let me know if you agree with my thoughts.